cheroot/test/test_makefile.py

```diff
@@ -0,0 +1,52 @@
+"""self-explanatory."""
```

Comment:
not really...

Reply:
I could write "Tests for the makefile module.", but I feel that's redundant to the name of this module. What would you recommend? Feel free to enhance it.

Reply:
BTW, I wouldn't have bothered writing a docstring at all were it not required by the linters.

Reply:
Yeah, I have a different policy now. I think it's useful to mention what to expect from the tests inside the module without forcing the reader to actually read tests.

Reply:
That sounds like a maintenance nightmare... trying to keep documentation about tests up-to-date with the code, especially when the documentation isn't adjacent to the code seems like you're just asking for trouble. As much as possible, I try to make code self-documenting and any documentation should expand on or clarify the purpose of a given piece of code and generally not restate what the code describes. Anyway, that's my approach. You're welcome to pursue more aggressive documentation strategies.

Reply:
I'm thinking more about something answering the WHY question. High-level, mostly not requiring updates over time. What this test suite is for, how it's different from others.